CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Added
+
+- `consecutiveOpenDoorHeartbeatCount` variable which tracks the number of heartbeats in a row a door has been left open (CU-86794et47).
+- `sendFallOffAlert` function to send an alert when the the consecutive count meets a certain threshold (CU-86794et47).
```

Comment:
function name should be a bit more descriptive. It's not guaranteed that the door sensor has fallen off. The alert just indicates that there has been no door closing activity. The most likely cause is that the magnet side of the door has been removed. I would suggest changing `sendFallOffAlert` to `sendDoorInactivityAlert`

Reply:
fixed in latest push!